lib/ex_unit/lib/ex_unit.ex

```diff
@@ -182,6 +182,8 @@ defmodule ExUnit do
           time = ExUnit.Server.modules_loaded()
           config = persist_defaults(configuration())
           %{failures: failures} = ExUnit.Runner.run(config, time)
+          Application.fetch_env!(:ex_unit, :after_suite)
+          |> Enum.each(fn fun -> fun.() end)
```

Comment:
Now that I think about it, maybe we should pass the result of `run` as argument to the function.

Reply:
Are you thinking that there could be conditional code in these callbacks that would, for example, only be executed if there are failures? I can see people that would want to do such things.

Reply:
Yeah, I can't think of any use cases right now, but it doesn't seem to be a weird thing to want.